### PR TITLE
Get endpoints for buildings and kingdom

### PR DESCRIPTION
**Pull request by rekagiday**
apiService for endpoints:
GET /[userId]/kingdom
GET /[userId]/kingdom/buildings
GET /[userId]/kingdom/buildings/[buildingId]

and mock responses in mockApiService

### CODE REVIEW

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/api/model/gameobject/Kingdom.java [hunk 1]
```diff
@@ -8,7 +8,7 @@
  * Created by User on 2017. 06. 14..
  */
 
-public class Kingdom extends BaseResponse{
+public class    Kingdom extends BaseResponse{
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/api/model/gameobject/Kingdom.java]: @rekagiday unnecessary spaces and space before opening bracket

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    private String type = "farm";
+    private int level = 1;
+    private int hp = 10;
+
+    @Override
+    public Call<BuildingsResponse> getBuildings(@Path("userId") int userId) {
+        return new MockCall<BuildingsResponse>() {
+            @Override
+            public void enqueue(Callback callback) {
+                callback.onResponse(null, Response.success(new BuildingsResponse(buildings)));
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java:47]: @rekagiday create a building list here with a few buildings

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java (hunk 2 above)
Comment (rekagiday) [reply app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java:47]: done

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java [hunk 3]
```diff
@@ -24,9 +25,12 @@
...
     private String name = "kingdomname";
     private long idOfUser = 1;
-    private List<Building> buildings = null;
-    private List<Resource> resources = null;
-    private List<Troop> troops = null;
+    private Building farm1 = new Building(1, "farm", 1, 2);
+    private Building mine1 = new Building(1, "mine", 1, 2);
+    private Building townhall = new Building(1, "townhall", 1, 10);
+    private List<Building> buildings = new ArrayList<>(Arrays.asList(farm1, mine1, townhall));
+    private List<Resource> resources = new ArrayList<>();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java]: @rekagiday dont skip the dummy resources :)

File: app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java [hunk 4]
```diff
@@ -24,9 +25,12 @@
...
+    private List<Troop> troops = new ArrayList<>();
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/tribesoflagopusandroid/api/service/MockApiService.java]: and troops

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**